tests/langchain/test_langchain_databricks_dependency_extraction.py

```diff
@@ -557,7 +557,8 @@ def mock_function_get(self, function_name):
     monkeypatch.setenv("DATABRICKS_TOKEN", "my-default-token")
     monkeypatch.setattr("databricks.sdk.service.catalog.FunctionsAPI.get", mock_function_get)
 
-    client = DatabricksFunctionClient()
+    with mock.patch("unitycatalog.ai.core.databricks.DatabricksFunctionClient.set_spark_session"):
+        client = DatabricksFunctionClient()
```

Comment:
```suggestion
    with mock.patch("unitycatalog.ai.core.databricks.DatabricksFunctionClient.set_spark_session"):
        client = DatabricksFunctionClient()
```

Let's add TODO comment.